recipes/numpy/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+cat > site.cfg <<'siteconfig'
+[DEFAULT]
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+
+[atlas]
+atlas_libs = openblas
+libraries = openblas
+
+[openblas]
+libraries = openblas
+library_dirs = $PREFIX/lib
+include_dirs = $PREFIX/include
+siteconfig
```

Comment:
So, we could do this and it once we are sure we tweaked it right it works. Though an equally effective strategy turns out to be just to create links as shown below. Though we should remove them when we are done.

```
ln -s /opt/conda2/lib/libopenblas.so /opt/conda2/lib/libblas.so
ln -s /opt/conda2/lib/libopenblas.so /opt/conda2/lib/liblapack.so
```

This works because NumPy and SciPy search for these by default and thus no configuration is required. Normally I have included this by default in the OpenBLAS recipe. However, as people have suggested having multiples BLASes (something I still disagree with), I have not done this in conda-forge's recipe so as to avoid clashing with other BLASes in this way.